overlord/snapstate/component.go

```diff
@@ -256,18 +256,18 @@ type componentInstallFlags struct {
 }
 
 type componentInstallTaskSet struct {
-	compSetupTaskID    string
-	beforeLink         []*state.Task
-	linkTask           *state.Task
-	postOpHookAndAfter []*state.Task
-	discardTask        *state.Task
+	compSetupTaskID     string
+	beforeLink          []*state.Task
+	linkTask            *state.Task
+	postOpHookToDiscard []*state.Task
```

Comment:
what does ToDiscard means here?

Reply:
The range of the tasks, [post-refresh/install hook -> discard). I'm happy to take suggestions on naming there.

Reply:
maybe all *state.Task field should have a name ending in Tasks, and all the []*state.Task ones, one ending in "Tasks" ?